src/global/globalNode.js

```diff
@@ -19,14 +19,15 @@ import Node from '../base/node';
 import FocusedNodeView from '../focused/focusedNodeView';
 
 class GlobalNode extends Node {
-  constructor (node) {
+  constructor (node, entryNode) {
     node.size = node.size || 120;
-    super(node, 'global');
+    super(node, 'global', entryNode);
     this.refreshLoaded();
   }
 
   isInteractive () {
-    return !this.isEntryNode();
+    return true;
```

Comment:
We could make that but if we have regions without child nodes user cannot drag them as well. And in general clicking shouldnt be connected with dragging nodes I guess